uber/site_sections/emails.py

```diff
@@ -16,18 +16,35 @@ def sent(self, session, **params):
     sent.restricted = [c.PEOPLE, c.REG_AT_CON]
 
     def pending(self, session, message=''):
-        approved = {ae.ident for ae in session.query(ApprovedEmail).all()}
+        automated_emails = []
+        last_job_completed = SendAllAutomatedEmailsJob.last_result.get('completed', False)
+        categories_results = SendAllAutomatedEmailsJob.last_result.get('categories', None)
+
+        count_query = session.query(Email.ident, func.count(Email.ident)).group_by(Email.ident)
+        sent_email_counts = {c[0]: c[1] for c in count_query.all()}
+
+        for automated_email in AutomatedEmail.instances.values():
+            category_results = categories_results.get(automated_email.ident, None) if categories_results else None
+            unsent_because_unapproved = category_results.get('unsent_because_unapproved', 0) if category_results else 0
```

Comment:
cool, I thought about that, but was also thinking in the future that we might have other metrics to extract from the category_results and it would be easier to keep them as two separate thoughts